### DISCUSSION TITLE
I would annotate item in the list with some text and tags

### DISCUSSION THREAD

**Discussion by MrBogomips** [community/community#51121] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

This should simplify the classification of starred repository to respect of many other dimensions.

For example, using a tag such as "Reference" or "Give a Try" and so on…

**Comment by signed** [comment]
I also prefer tagging favorites combined with searching / filtering based on a simple  tag expression language [like here](https://junit.org/junit5/docs/current/user-guide/#running-tests-tag-expressions) over plain list.

Usually I want to remember a favorite by multiple dimensions like  language (ruby, TypeScript, Java, ...), purpose (testing, orm, logging, ...), ... .
Currently I have a list for each dimension and add the repository to each of those lists.
